src/configure/configurers/remoteConfigurer.ts

```diff
@@ -0,0 +1,190 @@
+import * as fs from 'fs';
+import * as Path from 'path';
+import * as utils from 'util';
+import * as vscode from 'vscode';
+import { UserCancelledError } from 'vscode-azureextensionui';
+import { IProvisioningServiceClient } from "../clients/IProvisioningServiceClient";
+import { ProvisioningServiceClientFactory } from "../clients/provisioningServiceClientFactory";
+import { sleepForMilliSeconds } from '../helper/commonHelper';
+import { LocalGitRepoHelper } from '../helper/LocalGitRepoHelper';
+import { telemetryHelper } from '../helper/telemetryHelper';
+import { WizardInputs } from "../model/models";
+import { CompletePipelineConfiguration, DraftPipelineConfiguration, File, ProvisioningConfiguration } from "../model/provisioningConfiguration";
+import { Messages } from '../resources/messages';
+import { TelemetryKeys } from '../resources/telemetryKeys';
+import { TracePoints } from '../resources/tracePoints';
+import { IRemoteConfigurerBase } from "./remoteConfigurerBase";
+
+// tslint:disable-next-line:interface-name
+interface DraftFile {
+    content: string;
+    path: string;
+    absPath: string;
+}
+
+const Layer: string = "RemoteConfigurer";
+export class RemoteConfigurer implements IRemoteConfigurerBase {
```

Comment:
Should we name it to Provisioning Configurer?

Reply:
done